server/util/log/log.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 
+	gcplogging "github.com/buildbuddy-io/buildbuddy/server/util/log/gcp"
```

Comment:
Is this alias necessary? Could you refer to it as simply `gcp` or rename the package to `gcplog` if you think gcp is not descriptive?